server/admin/models.py

```diff
@@ -1,31 +1,41 @@
 from flask_login import current_user
-from flask_admin.contrib.sqla import ModelView
+from markupsafe import Markup
 
+from ..audit import AuditModelView
+from flask_admin.contrib.sqla import ModelView
 
 class UserView(ModelView):
     can_delete = True
+    can_view_details = True
     column_list = ["google_id", "first_name", "last_name", "email", "roles"]
     form_columns = ["first_name", "last_name", "email", "roles"]
 
 
-class IncidentView(ModelView):
+class IncidentView(AuditModelView):
     column_list = [
         "summary",
         "details",
         "types",
         "updated_on",
         "occurred_on",
         "reporter",
+        "audit_log_link",
     ]
     form_columns = ["summary", "details", "types", "occurred_on"]
 
+    def _audit_log_link(view, context, model, name):
+        return Markup(
+            f'<a href="/admin/auditlog/?record_id={model.id}">View Audit Logs</a>'
+        )
+
     def _render_reporter_link(view, context, model, name):
-        # if model.reporter:
-        #     return Markup('<img src="%s" />' % (model.reporter.profile_picture))
         return model.reporter.first_name
```

Comment:
This would be a handy place to link to the reporter's user details page.

This seems similar to `AuditLogView._user_link`. Could factor out. At its most generic something like a `render_model_link` formatter that links to details with `__str__` as linked text would probably cover a lot of the cross-linking you'll want here.